test/src/main/java/org/apache/accumulo/test/functional/SplitRecoveryIT.java

```diff
@@ -46,6 +46,8 @@
 import org.apache.accumulo.core.data.TableId;
 import org.apache.accumulo.core.data.Value;
 import org.apache.accumulo.core.dataImpl.KeyExtent;
+import org.apache.accumulo.core.fate.FateId;
+import org.apache.accumulo.core.fate.FateInstanceType;
```

Comment:
#4260 will conflict with these changes,  will hold off on merging it until this PR is merged.